nebu/tests/data/collection_no_changes/collection.xml

```diff
@@ -21,47 +21,47 @@
   <md:content-url>http://cnx.org/content/col11405/1.2</md:content-url>
   <md:content-id>col11405</md:content-id>
   <md:title>Intro to Computational Engineering: Elec 220 Labs</md:title>
-  
-  
+
+
   <md:version>1.2</md:version>
   <md:created>2011/05/24 10:31:56.888 GMT-5</md:created>
   <md:revised>2013/03/11 22:52:33.244 GMT-5</md:revised>
 
   <md:actors>
-    
-    
+
+
     <md:person userid="cavallar">
-      <md:firstname>Joseph</md:firstname>
+      <md:firstname>Hélène</md:firstname>
```

Comment:
TODO: Having changed this line probably had no effect whatsoever because metadata is not published nor used to detect what's changed 🤔, so I'll need to update the module's content instead.